### PR TITLE
Closes #101 SDTM.OE created for Ophtha

### LINKED ISSUE #101: Create OE for admiralophtha
See code https://github.com/pharmaverse/admiralophtha/blob/devel/dev/oe.R

@bms63 Should SDTM.OE be called `admiral_oe` or `admiralophtha_oe` as it is Ophtha specific.

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! 

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. 

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by millerg23** [comment]
@bms63 checks passed now. Also what am I doing wrong, when I look on board the Issue and pull request appear separately, but I appear to have them linked. Apologies if I am being a doofus ;-)

**Comment by bms63** [comment]
Hey @millerg23 we don't link the project board onto Pull Requests.  Since the issue is already up there on the board we just let that drive the board.

File: dev/oe.R [hunk 1]
```diff
@@ -0,0 +1,150 @@
+# OE
+
+library(dplyr)
+library(labelled)
```
Comment (bms63) [inline dev/oe.R:4]: Could you use `metatools` rather than labelled package please.  I'm trying to purge labelled from our scripts.  #87

**Review by bms63 — APPROVED** [review]
Looks great!!

**Pull request merged**